reference/catalog/categories_catalog.v3.yml

```diff
@@ -215,6 +215,11 @@ paths:
           description: 'Fields to exclude, in a comma-separated list. The specified fields will be excluded from a response. The ID cannot be excluded.'
           schema:
             type: string
+        - name: sort
+          in: query
+          description: 'Controls the sort order of the response. For example, `sort=name` sorts categories in alphabetical order by `name`.'
```

Comment:
I think we need to specify a list of all allowed values to sort by. In addition to `name` we have `id`, `sort_order` and `parent_id`

Reply:
Thanks @jyuricle , I added the allowed values